### PR TITLE
Bug 1175491 - Eliminate FB toggle in FTU until new version of FB integration arrives

### PR DESCRIPTION
**Pull request by fcampo**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0f5bfdd369572c895a9c2828c6d46a497bdef32c

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c8dfde7a1dbc99b1eaecfbb5d37dbcf6275eb992

File: apps/ftu/index.html [hunk 1]
```diff
@@ -19,7 +19,6 @@
   <link rel="stylesheet" href="style/style.css">
   <link rel="stylesheet" href="style/status.css">
   <link rel="stylesheet" href="style/overlay.css">
-  <link rel="stylesheet" href="style/fb_extensions.css">
   <link rel="stylesheet" href="style/tutorial.css">
```
Comment (sfoster) [inline apps/ftu/index.html]: If we aren't using the fb_extensions.css file anymore, we should remove it. There's also fb_* ids in style.css we can remove from the rule   
#import_contacts #fb_import, #import_contacts #fb_after_import2 {
  margin-top: 1rem;
}

File: shared/pages/import/js/oauth20.js [hunk 2]
```diff
@@ -47,7 +47,7 @@ if (typeof window.oauth2 === 'undefined') {
 
       // Enables simple access to test tokens for hacking up the app
       // This code will need to be deleted once we have a final product
-      fb.testToken = fb.testToken || parent.fb.testToken;
+      fb.testToken = fb.testToken || (parent.fb && parent.fb.testToken);
```
Comment (borjasalguero) [inline shared/pages/import/js/oauth20.js]: Do we need to fix this?

Comment (fcampo) [inline shared/pages/import/js/oauth20.js]: yeah, it triggers a null parent error (fb is used by gmail, don't ask me why), so we better check that parent exists first. 
other option would be to fully refactor the fb.token, but I'd leave that for future facebook related bugs (too complex for doing it here, where we just want to hide the toggle)

File: apps/ftu/index.html [hunk 3]
```diff
@@ -441,28 +434,6 @@ <h2 id="time-zone-title">
...
-            </li>
-            <li id="fb_after_import1" class="hidden">
-              <p data-l10n-id="importFb2">Facebook</p>
-              <p id="fb_import_feedback"></p>
-            </li>
-            <li class="importOption hidden" id="fb_after_import2">
-              <p data-l10n-id="afterFbImport"></p>
-            </li>
-          </ul>
-        </article>
```
Comment (borjasalguero) [inline apps/ftu/index.html]: Do we need [1] and all panels related with FB? Is `ServicesLauncher` needed anymore?

[1] https://github.com/fcampo/gaia/blob/fb_out_ftu_1175491/apps/ftu/index.html#L751

File: apps/ftu/index.html [hunk 4]
```diff
@@ -441,28 +434,6 @@ <h2 id="time-zone-title">
...
-              <button id="fb-import-button" class="icon icon-fb" data-l10n-id="importFb2">
-                  Facebook
-              </button>
-              <p id="no-network" class="hidden" data-l10n-id="noConnection11">
-                To import Facebook friends connect to Wi-Fi or data network
-              </p>
-            </li>
-            <li id="fb_after_import1" class="hidden">
-              <p data-l10n-id="importFb2">Facebook</p>
-              <p id="fb_import_feedback"></p>
```
Comment (borjasalguero) [inline apps/ftu/index.html]: Remove l10n strings as well, as all of them are not going to be used anymore.

File: apps/ftu/index.html (hunk 1 above)
Comment (fcampo) [inline apps/ftu/index.html]: actually that was my mistake, we do need them, but renaming to avoid 'fb' connection

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3ad290067b7a5ef80d77d3279bc6913fbf2b1204

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4ef837a5ae02cc7dc24fa53f386a59038a5da20f

**Pull request merged**